client/apps/articles_list/client/client.coffee

```diff
@@ -10,7 +10,7 @@ sd = require('sharify').data
 request = require 'superagent'
 icons = -> require('../icons.jade') arguments...
 
-module.exports = ArticlesListView = React.createClass
+module.exports.ArticlesListView = ArticlesListView = React.createClass
```

Comment:
Ah cool -- I think they were removed when the displayName/new react naming was added to them, but didn't realize they added this utility.